tests/test_load_save_keys.py

```diff
@@ -17,10 +17,13 @@
 """Unittest for saving and loading keys."""
 
 import base64
-import unittest
+import mock
```

Comment:
We need compatibility with Python 2, and `unittest.mock` doesn't provide that to us. According to [`mock` documentation](https://mock.readthedocs.io/en/latest/):

> mock is now part of the Python standard library, available as unittest.mock in Python 3.3 onwards. However, if you are writing code that runs on multiple versions of Python the mock package is better, as you get the newest features from the latest release of Python available for all Pythons.